draft-ietf-quic-ack-frequency.md

```diff
@@ -340,6 +340,19 @@ see {{out-of-order}}.
 
 {{congestion}} and {{batch}} describe exceptions to this strategy.
 
+All packets still have to be acknowledged at least once with this extension,
+as stated in {{Section 13.2.1 of QUIC-TRANSPORT}}. With large values for
+Ack-Eliciting Threshold or the Reordering Threshold,
+implementations might accumulate multiple new ACK ranges before sending an ACK.
+As such, implementations have to take more care to not truncate ACK ranges
+before they were sent at least once in an ACK frame.
+Further, as discussed in {{Section 13.2.4 of QUIC-TRANSPORT}} this does not
+guarantee that every acknowledgment is seen by the sender before it is no
+longer included in the ACK frame. Therefore, when ACK frames are sent less often,
```

Comment:
```suggestion
As such, implementations have to take more care to avoid truncating ACK ranges
before they are sent at least once. As discussed in {{Section 13.2.4 of QUIC-TRANSPORT}}
this does not guarantee that every acknowledgment is seen by the sender.
Therefore, when ACK frames are sent less often,
```